apps/user-office-backend/src/middlewares/factory/pdf.ts

```diff
@@ -33,12 +33,23 @@ router.get(`/${PDFType.PROPOSAL}/:proposal_pks`, async (req, res, next) => {
       collectionFilename: `proposals_${getCurrentTimestamp()}.pdf`,
       singleFilename: '',
     };
+    const options = ((queryParams) => {
+      return {
+        filter: queryParams.filter?.toString(),
+        pdfTemplateId:
+          req.query?.pdfTemplateId &&
+          !isNaN(+req.query?.pdfTemplateId?.toString())
+            ? +req.query?.pdfTemplateId?.toString()
```

Comment:
```suggestion
            ? +req.query.pdfTemplateId.toString()
```

Reply:
? check is not needed, as it is already being passed. 

Also, i feel the function toString() is being called two times and can be reduced to one. Please take a look, if you agree with me.